ConsoleGUI/Controls/ListView.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ConsoleGUI.Data;
+using ConsoleGUI.Input;
+using ConsoleGUI.UserDefined;
+
+namespace ConsoleGUI.Controls
+{
+    public class ListView : SimpleControl, IInputListener
+    {
+        public Color TextColor { get; set; } = Color.White;
+        public Color SelectedTextColor { get; set; } = Color.Black;
+
+        private int selectedIndex = -1;
+        private int totalCount = -1;
+
+        public ListView()
+        {
+            Content = new VerticalScrollPanel()
+            {
+                Content = new VerticalStackPanel()
+            };
+        }
+
+        public string SelectedText
+        {
+            get
+            {
+                try
+                {
+                    var selectedItem = ((Content as VerticalScrollPanel).Content as VerticalStackPanel).Children.ElementAt(selectedIndex);
```

Comment:
You should assign those internal containers to `readonly` fields in the constructor. This way you wouldn't have to use `as` all the time and the code would be simpler.